src/main/java/racingcar/domain/RacingCar.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.domain;
+
+public class RacingCar {
+
+    private static final int NAME_MAX_LENGTH = 5;
+    private static final int DEFAULT_POSITION = 0;
+
+    private final String name;
+    private int position = DEFAULT_POSITION;
+
+    public RacingCar(String name) {
+        isEmpty(name);
+        checkNameLength(name);
+        this.name = name;
+    }
+
+    private void isEmpty(String name) {
+        if (name == null || name.isEmpty()) {
+            throw new IllegalArgumentException("자동차 이름은 비어있을 수 없다.");
+        }
+    }
+
+    private void checkNameLength(String name) {
+        if (name.length() > NAME_MAX_LENGTH) {
+            throw new IllegalArgumentException("자동차 이름은 5자를 초과할 수 없다.");
+        }
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void moveByCommand(boolean command) {
+        if (command) {
+            position++;
+        }
+    }
```

Comment:
매개변수 boolean 타입이 `command`라는 이름이 때문일까요? 코드가 조금 모호하네요.

```suggestion
    public void move() {
            position++;
    }
```
이런 형태를 갖고, 도메인를 호출하는코드에서 조건을 분기하는 건 어떻게 상각하세요?